joomla/joomla-cms
# [4.0] Toolbarfix

Pull Request for Issue # https://github.com/joomla/joomla-cms/issues/28036.

### Summary of Changes
I removed the override for the `Toolbar` in Autum Template that was created in https://github.com/joomla/joomla-cms/pull/27918

In this override the filter was moved before the search int the toolbar. I hope that I not oversee something again. But I see no reason for doing this move. That is why for me the easiest solution is to delete this override.


### Testing Instructions

Please see https://github.com/joomla/joomla-cms/pull/28032 and test the view on Mobil.

### Expected result



### Actual result



### Documentation Changes Required
